pyaedt/edb_core/primitives.py

```diff
@@ -0,0 +1,92 @@
+"""
+This module .
+"""
+
+from pyaedt import aedt_exception_handler
+from pyaedt.edb_core.layout import EdbLayout
+from pyaedt.edb_core.padstack import EdbPadstacks
+
+class Trace(object):
+    """
+    Manage EDB trace
+    """
+    def __init__(self, width="0.0", point_list=None, layer="", net_name=""):
+        if point_list is None:
+            point_list = [[0.0, 0.0], [1e-3, 0.0]]
+        self.point_list = point_list
+        self.width = width
+        self.layer = layer
+        self.net_name = net_name
+        self.layout_core = EdbLayout(self)
+
+    @aedt_exception_handler
+    def place_line(self):
```

Comment:
do you seriously need a class for a single function?

why just not to use a function?

Reply:
same was committed by mistake will remove it

Reply:
btw this PR has been dropped and was never planned to be merged. I will cancel it